test/in-browser/ts/test-utils.ts

```diff
@@ -12,9 +12,12 @@ const prettier = require("prettier/standalone");
 const plugins = [require("prettier/parser-html")];
 
 export const fetchFixtureHtml = async fixture => {
-  const fixtureBaseUrl = "http://0.0.0.0:4000";
+  const fixtureBaseUrl = "http://0.0.0.0:4000/fixtures";
```

Comment:
Just a question. Is the `fixtures` in the path intended here? Asking because you replaced `fixtures` with `locally-hosted-files` in the other file.

Reply:
Yes it is international and correct :)